moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -53,11 +53,36 @@
 from pathlib import Path
 from typing import Optional, List
 import logging
+import re
 from ament_index_python.packages import get_package_share_directory
 
 from launch_param_builder import ParameterBuilder, load_yaml, load_xacro
 
 
+moveit_configs_utils_path = Path(get_package_share_directory("moveit_configs_utils"))
+
+
+def get_pattern_matches(folder, pattern):
```

Comment:
Why not use [glob](https://docs.python.org/3/library/glob.html) here?

Reply:
Because this method returns the proper group (i.e. the part before `_controllers.yaml`). So either I have this custom method OR I use glob, then have a separate regex for grabbing the group. It's messy.